projects/vdk-plugins/vdk-notebook/tests/test_plugin.py

```diff
@@ -1,30 +1,64 @@
 # Copyright 2021 VMware, Inc.
 # SPDX-License-Identifier: Apache-2.0
 import os
+import unittest
 from unittest import mock
 
 from click.testing import Result
 from vdk.plugin.notebook import notebook_plugin
+from vdk.plugin.sqlite import sqlite_plugin
 from vdk.plugin.test_utils.util_funcs import cli_assert_equal
 from vdk.plugin.test_utils.util_funcs import CliEntryBasedTestRunner
 from vdk.plugin.test_utils.util_funcs import jobs_path_from_caller_directory
 
-"""
-This is a sample test file showing a recommended way to test new plugins.
-A good way to test a new plugin is how it would be used in the command that it extends.
-"""
 
+@mock.patch.dict(
+    os.environ,
+    {
+        "VDK_DB_DEFAULT_TYPE": "SQLITE",
+        "VDK_INGEST_METHOD_DEFAULT": "sqlite",
+    },
+)
+class JupyterTests(unittest.TestCase):
+    def setUp(self) -> None:
+        self.__runner = CliEntryBasedTestRunner(notebook_plugin, sqlite_plugin)
 
-def test_dummy():
-    with mock.patch.dict(
-        os.environ,
-        {
-            # mock the vdk configuration needed for our test
-        },
-    ):
-        # CliEntryBasedTestRunner (provided by vdk-test-utils) gives a away to simulate vdk command
-        # and mock large parts of it - e.g passed our own plugins
-        runner = CliEntryBasedTestRunner(notebook_plugin)
+    def test_notebook_plugin(self) -> None:
```

Comment:
what's the purpose of this test? is it the happy path or success scenario?

Reply:
it is the success  scenario, I might rename the test